CONTRIBUTING.md

```diff
@@ -72,28 +70,31 @@ ____________________________________O/_______
 Estuary stores data on IPFS before replicating it to the Filecoin Network. When you store data using Estuary, that data will first go to an `estuary-shuttle` *node* that utilizes IPFS as *hot-storage* before replication to the Filecoin Network begins.
 
 Build the shuttle binary
-
 ```bash
 $ make estuary-shuttle
 ```
 
-Then, initialize a shuttle node using the Estuary API endpoint `/admin/shuttle/init` or the admin UI at <host>/admin/shuttle
-
+Then, initialize a shuttle node. You can do this by using the admin UI at `<host>/admin/shuttle`, by posting to the Estuary API endpoint `/admin/shuttle/init` 
```

Comment:
nit: `, or by posting to the Estuary API en` - you missed "or"

Reply:
it's a list of 3 with or at the end :)